src/modules/token/fungible/hooks/useAmountField.js

```diff
@@ -64,23 +64,36 @@ const useAmountField = (initialValue, balance, token) => {
   }, [initialValue]);
 
   const onAmountInputChange = ({ value }, maxAmount) => {
```

Comment:
can we use `amount`? that way its more readable, i.e `({ value: amount }, maxAmount)`